runtime/reset.go

```diff
@@ -23,20 +23,12 @@ import (
 )
 
 func (d *Default) reset(cluster *v1.Cluster) error {
-	err := d.resetNodes(cluster.Spec.Nodes.IPList)
-	if err != nil {
-		logger.Error("failed to clean nodes %v", err)
-	}
-	err = d.resetMasters(cluster.Spec.Masters.IPList)
-	if err != nil {
-		logger.Error("failed to clean masters %v", err)
-	}
+	d.resetNodes(cluster.Spec.Nodes.IPList)
```

Comment:
If reset node failed, we don't know what happen. Only print log and not return error because we want to continue the clean steps after reset.

Reply:
but the fact is that the function `resetNodes` never returns a non-nil error.